### PR TITLE
Fix BackendServices functionality to reflect standard functionality

### PR DESCRIPTION
**Pull request by ihmccreery**
- `get` should return nil when finding nothing
- fix `create` and `destroy`
- fix tests

### CODE REVIEW

**Comment by ihmccreery** [comment]
@plribeiro3000 can I get a :+1:?

File: lib/fog/google/models/compute/backend_services.rb [hunk 1]
```diff
@@ -13,8 +13,12 @@ def all(filters={})
         end
 
         def get(identity)
+          resonse = nil
```
Comment (plribeiro3000) [inline lib/fog/google/models/compute/backend_services.rb]: Why this?

**Comment by plribeiro3000** [comment]
@ihmccreery sure. made a small comment. It seems to be a typo but i'm not sure the real intent behind it if it is corrected since the next line overwrite it anyway.

**Comment by ihmccreery** [comment]
Good catch.  That's actually a typo [here](https://github.com/fog/fog-google/commit/1976f5c62027042311809e5d17de36bfec833dcb#L16) as well, which I'll fix.

**Comment by ihmccreery** [comment]
*This was merged.*  Still getting used to the rebase, sorry.

**Comment by plribeiro3000** [comment]
Don't worry. it happens to us all. =)

**Pull request closed without merging**